.github/workflows/build/Dockerfile.ubuntu-2004

```diff
@@ -49,6 +49,8 @@ RUN pip3 install -U \
     msgpack-python==0.5.6 \
     orderedset==2.0.3 \
     packaging==20.9 \
+    Pygments==2.2.0 \
+    Pympler==0.8 \
     pip==9.0.3 \
```

Comment:
As of #1548 a custom pip version is not required, but importlib_metadata is required

Reply:
@andrewwhitehead, thanks for highlighting this. I created the following issues for the `ubuntu-20.04-upgrade` feature branch #1549